lightly_studio/src/lightly_studio/api/server.py

```diff
@@ -101,6 +102,9 @@ def _is_port_available(host: str, port: int) -> bool:
 
     for family in families:
         with socket.socket(family, socket.SOCK_STREAM) as s:
+            # Allow port binding during TIME_WAIT to avoid false "port busy" checks.
```

Comment:
In case anyone is interested in reading up. I found this link helpful: https://serverframework.com/asynchronousevents/2011/01/time-wait-and-its-design-implications-for-protocols-and-scalable-servers.html